### PR TITLE
add watering can code

### PR DESCRIPTION
**Pull request by kanan42**
uses the water can to clean the pet

### CODE REVIEW

**Comment by Donnyd88** [comment]
It looks like the elif you added is duplicated. Do we need it twice? Otherwise looks good.

**Comment by dinaolmelak** [comment]
Wait, which elif? Can you comment on the file next to the line number

**Comment by dinaolmelak** [comment]
Oh yeah you're right, I see it

File: game_util/scene_items.py [hunk 1]
```diff
@@ -119,6 +119,12 @@ def get_collision_item(self):
         elif self.item_id == config.ItemID.full_cup and self.rect.collidepoint(self.interacting_pet.get_location()):
             self.interacting_pet.set_current_animation(config.RockActions.drinking.value, True)
             return config.ItemID.full_cup
+        elif self.item_id==config.ItemID.watering_can and self.rect.collidepoint(self.interacting_pet.get_location()):
+            self.interacting_pet.set_current_animation(config.RockActions.clean.value,True)
+            return config.ItemID.watering_can
+        elif self.item_id==config.ItemID.watering_can and self.rect.collidepoint(self.interacting_pet.get_location()):
```
Comment (dinaolmelak) [inline game_util/scene_items.py:125]: Yeah this elif is duplicated

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
Just remove one `elif` and do another commit and push to this branch 😃

File: game_util/scene_items.py (hunk 1 above)
Comment (kanan42) [reply game_util/scene_items.py]: resolved

**Review by dinaolmelak — APPROVED** [review]
LGTM

**Review by RuthAnn1 — APPROVED** [review]
Looks good

**Pull request merged**